doc/source/tutorial/linalg.rst

```diff
@@ -956,3 +956,11 @@ Van der Monde         `numpy.vander`                     Create a Van der Monde
 
 
 For examples of the use of these functions, see their respective docstrings.
+
+Advanced Features
+-----------------
+
+.. _linalg-batch:
+
+Batch Support
+^^^^^^^^^^^^^
```

Comment:
Probably will just link to the notebook. Ideally we'd convert all of `linalg.rst` to markdown, maybe add a TOC rather than a single masssive page. We could add another section here about the `overwrite_` inputs and include a link in the standard note where applicable.